src/main/java/org/testeditor/fixture/web/WebDriverFixture.aml

```diff
@@ -0,0 +1,136 @@
+package org.testeditor.fixture.web
```

Comment:
We should write a ticket that will correct/adjust the templates according to the conventions we decided on: [http://jira.system.local/browse/OTA-370]